internal/store/engine_state.go

```diff
@@ -250,8 +250,10 @@ type ManifestState struct {
 	// The last `BuildHistoryLimit` builds. The most recent build is first in the slice.
 	BuildHistory []model.BuildRecord
 
-	// If the pod isn't running this container then it's possible we're running stale code
-	ExpectedContainerID container.ID
+	// If a container has been updated in-place, then its contents have diverged
+	// from the image it's built on.
+	LiveUpdatedContainerID container.ID
```

Comment:
i wonder if this is still usefully called "ExpectedContainerID", since its use in code is "we expect to see container(s) X: if we don't, something is wrong."

I might comment this field like:
> The container ID that we've run a LiveUpdate on, if any. Its contents have diverged from the image it's built on. If this container doesn't appear on the pod, we've lost that state and need to rebuild.

Reply:
description added.

The name "ExpectedContainerID" is going to be incorrect once we have multiple container IDs, some of which may not have been live updated yet.

Reply:
I mean, container IDs will only get added to this list once they've been live updated. But sure.